ProEnchanters.lua

```diff
@@ -1197,6 +1199,190 @@ ProEnchanters.UIElements = {}
 ProEnchanters.frame = CreateFrame("Frame")
 ProEnchantersSettings = ProEnchantersSettings or {}
 
+-- Tooltips for Enchants
+local function tooltipFormat(enchValue)
+	if mouseFocus ~= "" then
+		GameTooltip:ClearLines()
+	end
+
+	if mouseFocus == "enchantButton1" then
```

Comment:
there was a naming issue with 5 "enchantButton" so i added a number instead of fixing it